test/createCycleMiddleware.test.js

```diff
@@ -54,9 +60,9 @@ describe('Redux cycle middleware', () => {
       const pong$ = sources.ACTION
         .filter(action => action.type === 'PING')
         .map(() =>
-            xs.periodic(10000)
-                .take(1)
-                .mapTo({ type: 'PONG' })
+          xs.periodic(10000)
```

Comment:
I've asked this before, but why not changing this now and importing `delay` from the extra operators?